tests/Services/AdSlotServiceTest.php

```diff
@@ -101,6 +101,17 @@ public function testIsUpdateableViaResourceInstance()
         $this->assertInstanceOf(AdSlot::class, $adSlot);
     }
 
+    public function testUpdateExternalUrl()
+    {
+        $adSlot = $this->service->find(Fixtures::adSlotId());
+        $gamService = $this->createService('gam');
+        $accountId = 123456;
+        $externalUrl = $gamService->getGamUrl($adSlot->refId, $accountId);
+        $adSlot->externalUrl = $externalUrl;
+        $adSlot = $this->service->update($adSlot);
+        $this->assertEquals($adSlot->externalUrl, $externalUrl);
```

Comment:
@ashishkumarqait won't this fail now that we are getting an unencoded url at the end of getGamUrl but I'm guessing it's encoded when we do `service->update($adSlot)` for `adSlot->externalUrl`?

Reply:
No this works fine, tried and tested